site/src/api.rs

```diff
@@ -391,6 +391,94 @@ pub mod status {
     }
 }
 
+pub mod status_new {
+    use chrono::{DateTime, Utc};
+    use database::BenchmarkSet;
+    use serde::Serialize;
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkRequestStatusUi {
+        pub state: String,
+        pub completed_at: Option<DateTime<Utc>>,
+        pub duration: Option<u32>,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkRequestTypeUi {
+        pub r#type: String,
+        pub tag: Option<String>,
+        pub parent_sha: Option<String>,
+        pub pr: Option<u32>,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkRequestUi {
+        pub status: BenchmarkRequestStatusUi,
+        pub request_type: BenchmarkRequestTypeUi,
+        pub commit_date: Option<DateTime<Utc>>,
+        pub created_at: DateTime<Utc>,
+        pub backends: Vec<String>,
+        pub profiles: String,
+        pub errors: Vec<String>,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkJobStatusUi {
+        pub state: String,
+        pub started_at: Option<DateTime<Utc>>,
+        pub completed_at: Option<DateTime<Utc>>,
+        pub collector_name: Option<String>,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkJobUi {
+        pub target: String,
+        pub backend: String,
+        pub profile: String,
+        pub request_tag: String,
+        pub benchmark_set: BenchmarkSet,
+        pub created_at: DateTime<Utc>,
+        pub status: BenchmarkJobStatusUi,
+        pub deque_counter: u32,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct BenchmarkInProgressUi {
+        pub request: BenchmarkRequestUi,
+        pub jobs: Vec<BenchmarkJobUi>,
+    }
+
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct CollectorConfigUi {
```

Comment:
You don't really even need the `Ui` suffixes, since the structs are in a separate module.